### DISCUSSION TITLE
R3 and L3 directions

### DISCUSSION THREAD

**Discussion by ve1la** [nirgoren/Eddienput#47] [Technical support]
Hello, is there a way to map R3 and L3 to various directions
By example dpad got DPAD_UP, DPAD_DOWN_RIGHT etc
I want the same thing for the two analog sticks

Also I'd love if I could chose by exactly how much bend the analog like in dolphin tas tools rather than have only 8 options https://i.imgur.com/fA7b0sO.png

**Comment by nirgoren** [comment]
Right now this is not possible. This functionality may be added in the future if there is enough demand.

How exactly would you like this feature to work? Syntax wise etc.

Code contributions are always welcome.

**Comment by ve1la** [comment]
So sorry for the late reply, I totally forgot
Unfortunately I don't know coding myself, wish I could contribute

Something like L_Stick_X70Y50 would lead to the analog be in this position https://i.imgur.com/eoJvn2k.png
